bokeh/models/glyphs.py

```diff
@@ -893,6 +926,39 @@ class Text(Glyph):
     The %s values for the text.
     """)
 
+class VBar(Glyph):
+    """ Render vertical bars, given a center coordinate, width and (top, bottom) coordinates. """
+
+    __example__ = "tests/glyphs/VBar.py"
+
+    # a canonical order for positional args that can be used for any
+    # functions derived from this class
+    _args = ('x', 'width', 'top', 'bottom')
+
+    x = NumberSpec(help="""
+    The x-coordinates of the centers of the bars.
+    """)
+
+    width = NumberSpec(help="""
+    The widths of the vertical bars.
+    """)
+
+    bottom = NumberSpec(help="""
+    The y-coordinates of the bottom edges.
+    """)
```

Comment:
I'm not suggesting adding a default for `top`.  I expect `VBar` will be used to easily create bar-like charts using `bokeh.plotting`. The suggestion is merely to make what assume is the common case -- bars starting from `y=0` slightly easier to spell.
```
plot.vbar(x=[1, 2, 3], width=0.5, top=[1,2,3])
```
instead of 
```
plot.vbar(x=[1, 2, 3], width=0.5, bottom=0, top=[1,2,3])
```
It's s minor thing, but common cases are the most important places for minor improvements. Of course if the bars don't start at zero the user can still supply `bottom` explicitly, that's no different.